google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/RandomResultSetGenerator.java

```diff
@@ -39,7 +39,7 @@
  * of Cloud Spanner filled with random data.
  */
 public class RandomResultSetGenerator {
-  private static Type[] generateTypes(Dialect dialect) {
+  public static Type[] generateAllTypes(Dialect dialect) {
```

Comment:
Is this method meant to be used publicly? If not, can we have it annotated as `VisibleForTesting`, if it's meant to be used only in tests?

Reply:
It is indeed only intended for testing, but I don't think we need to add the annotation in this case, as it is only defined in `src/test/java`, meaning that it will also only be available to test code.